pkgs/development/python-modules/progressbar2/default.nix

```diff
@@ -23,17 +15,19 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [ python-utils ];
-  nativeBuildInputs = [ pytestrunner ];
-  checkInputs = [
-    pytest sphinx flake8 pytest-flakes pytestcov
-    pytestcache freezegun
+
+  # depends on unmaintained pytest-pep8
+  # https://github.com/WoLpH/python-progressbar/issues/241
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "progressbar"
+    "progressbar.shortcuts"
```

Comment:
Does progressbar by chance import all the other ones?

Reply:
it does actually